images/win/scripts/Installers/Install-Vsix.ps1

```diff
@@ -0,0 +1,27 @@
+###################################################################################

```

Comment:
You should also update templates to add this file to templates and remove `Install-SSDTExtensions.ps1` from templates. I suggest to run it right after VS installation

Reply:
Right.
Done